modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -303,6 +323,30 @@ You can specify values for the following parameters of the API Console Request P
 *Headline Font Family (Optional)*:: Specifies the headline font family.
 *Error Color (Optional)*:: Specifies the color of error messages.
 
+To enable users to select an application to populate the `clientId` and `clientSecret` credentials, add credential annotations to the application specification:
```

Comment:
"credentials" = "credential fields"? "credential values"?

Reply:
Good catch. Updated to "credential values".